specification/cdn/resource-manager/Microsoft.Cdn/preview/2024-06-01-preview/examples/TunnelPolicies_Update.json

```diff
@@ -0,0 +1,91 @@
+{
+  "parameters": {
+    "api-version": "2024-06-01-preview",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "tunnelPolicyName": "tunnelPolicy1",
+    "tunnelPolicyUpdateProperties": {
+      "properties": {
+        "tunnelType": "HttpConnect",
+        "domains": [
+          {
+            "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/customdomains/testdomain1"
+          },
+          {
+            "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/customdomains/testdomain2"
+          }
```

Comment:
do you plan to add linked access checks to ensure that the caller has permissions to access these domains.

Reply:
@barathvg could you please answer this question?

Reply:
waiting for the response here.

Reply:
@raosuhas yes linked access checks are added in the manifest.

Reply:
Here is the Snippet:
{
      "name": "profiles/tunnelpolicies",
      "commonApiVersions": [        
        "2024-06-01-preview"
      ],
      "routingType": "ProxyOnly",
      "resourceDeletionPolicy": "Cascade",
      "linkedAccessChecks": [
        {
          "actionName": "Microsoft.Cdn/profiles/tunnelpolicies/write",
          "linkedProperty": "properties.domains[*].id",
          "linkedActionVerb": "write"
        },
        {
          "actionName": "Microsoft.Cdn/profiles/tunnelpolicies/write",
          "linkedProperty": "properties.targetgroups[*].id",
          "linkedActionVerb": "write"
        }
      ],